src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukButton.scala.html

```diff
@@ -41,8 +41,10 @@
 
 @commonAttributes(classNames: String) = {class="@classNames" data-module="govuk-button"@toAttributes(attributes)}
 
+@hasPreventDoubleClick = @{preventDoubleClick.isDefined}
+
 @buttonAttributes = {
-  @name.mapNonEmpty {name => name="@name"}@if(disabled) { disabled="disabled" aria-disabled="true"}@if(preventDoubleClick) { data-prevent-double-click="true"}
+  @name.mapNonEmpty {name => name="@name"}@if(disabled) { disabled="disabled" aria-disabled="true"}@if(hasPreventDoubleClick) { data-prevent-double-click="@{preventDoubleClick.get}"}
```

Comment:
this might work, though have to check that it adds spaces before attribute name correctly. As an aside it's interesting to see that previously if it was `false` then we wouldn't output any attribute and I guess it would use the default, where as now if it's set to false we'll set an attribute. Seems like the old behaviour could have meant (if default ever changed to true) that there was no way to turn it off 😅 
```suggestion
  @name.mapNonEmpty {name => name="@name"}@if(disabled) { disabled="disabled" aria-disabled="true"}@preventDoubleClick.map {value => data-prevent-double-click="@value"}
```